src/App/dtml/copyright.dtml

```diff
@@ -6,16 +6,16 @@
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
 
   <title>Zope Public License (ZPL) Version 2.1</title>
-  <link rel="stylesheet" type="text/css" href="/++resource++zmi/bootstrap-4.1.1/bootstrap.min.css" />
-  <link rel="stylesheet" type="text/css" href="/++resource++zmi/zmi_base.css" />
+  <link rel="stylesheet" type="text/css" href="&dtml-BASEPATH1;&dtml-BASEPATH1;/++resource++zmi/bootstrap-4.1.1/bootstrap.min.css" />
```

Comment:
Why are you using `&dtml-BASEPATH1;` here two times?

Reply:
@icemac Done.